testutil/tx/cosmos.go

```diff
@@ -0,0 +1,87 @@
+package tx
```

Comment:
was there an issue with creating it in the `testutil` package?

Reply:
no issues